route/build/src/histVars_data.f90

```diff
@@ -148,7 +163,7 @@ SUBROUTINE aggregate(this,             & ! inout:
         write(message,'(2A,G0,A,G0)') trim(message),'history buffer reach size:',this%nRch,'/= input data reach size:',nRch_input
         ierr=81; return
       end if
-
+! need to convert double precision to single precision??
```

Comment:
Remove this comment you just need to change the definition of the output file precision to output in single/double.

Reply:
removed.